arithmetics/arithmetics.py

```diff
@@ -0,0 +1,10 @@
+# Some arithmetics exercises
+a = 5
+b = 3
+print('Addition: ', a + b)
+print('Subtraction: ', a - b)
+print('Multiplication: ', a * b)
+print('Division (float): ', a / b)
+print('Division (floor): ', a // b)
+print('Modulus: ', a % b)
+print('Exponent: ', a ** b)
```

Comment:
According to PEP 8, a newline at the end of a file is considered good practice. This newline is not a blank line; it's just the presence of a line break at the end of the file. Including this newline can help avoid issues with some tools that may not handle files without a newline properly.

PEP 8 is the style guide for Python code, and it includes recommendations for how to format your code to make it more readable and consistent.